### ISSUE TITLE
Nice work!

### ISSUE THREAD

**Issue by robksawyer** [jbentley10/header-in-nextjs#1]

**Comment by jbentley10** [comment]
Definitely! It has been a while since I've done a project in React like this. The flow of being able to make changes and see them immediately refresh in the browser was amazing. Also loved being able to merge into the master branch and have changes be deployed right away.
